src/utils/createXArrayWithStep.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Create an array with numbers starting from "from" with step "step" of length "length"
+ *
+ * @param options - options
+ * @return - array of floats
+ */
+export function createXArrayWithStep(
+  options: {
+    /**
+     * start value of range
+     * @default 0 */
+    from?: number;
+    /**
+     * step if "to" not defined
+     * @default 1
+     */
+    step?: number;
+    /**
+     * number of points in range
+     * @default 1000 */
+    length?: number;
+    /**
+     * include from
+     * @default true */
+    includeFrom?: boolean;
```

Comment:
Those 3 properties should not exist if we specify step.

And step should not exist in the other method.